client/system_version_test.go

```diff
@@ -30,11 +30,11 @@ func TestSystemVersion(t *testing.T) {
 		if !strings.HasPrefix(req.URL.Path, expectedURL) {
 			return nil, fmt.Errorf("Expected URL '%s', got '%s'", expectedURL, req.URL)
 		}
-		version := types.SystemVersion{
+		versionConfig := types.SystemVersion{
```

Comment:
I do not think this is a config, version is just OK for me.

Reply:
The config stuff is only for post request, right?

Reply:
Yes, config is usually stored in the post request body. While this part of code is simulating a server which is returning a http response. And for this API, it returns `types.SystemVersion`.